src/registers/model_specific.rs

```diff
@@ -71,6 +71,10 @@ pub struct SCet;
 #[derive(Debug)]
 pub struct Pat;
 
+/// IA32_APIC_BASE: status and location of the local APIC
+#[derive(Debug)]
+pub struct Apic;
```

Comment:
Let's rename this to `ApicBase`. I realize that the MSR contains more than just the base address, but AMD's and Intel's manuals both use the name and I think we should be consistent here.

Let's do the same for `ApicFlags`.